client/my-sites/email/email-management/gsuite-users-card/index.jsx

```diff
@@ -63,17 +63,18 @@ class GSuiteUsersCard extends React.Component {
 	};
 
 	renderDomain( domain, users ) {
+		const license = users[ 0 ].product_name;
```

Comment:
I'm wondering whether a comment explaining that the product name is going to be the same for each user would be useful here. Probably not that much I guess.

Reply:
Certainly makes sense